src/test/functional/features/email-addresses.feature

```diff
@@ -13,6 +13,9 @@ Feature: Email-addresses
     Then I can view the existing emails
 
   Scenario: Add new Email Addresses
+    # Clear out potential left-over emails from the previous run before adding new ones
```

Comment:
Do we wanna remove this commented out test?

Reply:
It is not commented out :-) It is a comment to explain why we need to remove all existing email entries before adding a new ones. We can remove the comment if you think it is redundant. It is necessary to remove first because sometimes the previous test run fails to complete and causing some email entries getting left behind. When this happens the new duplication validation will fail because the entry in the new test duplicate with the entry on the previous run.